package/kernel/mac80211/realtek.mk

```diff
@@ -359,3 +370,83 @@ define KernelPackage/rtl8723bs/description
  on the 1st gen Intel Compute Stick, the CHIP and many other Intel Atom and ARM
  based devices.
 endef
+
+define KernelPackage/rtw89/config
+	config PACKAGE_RTW89_DEBUG
+		bool "Realtek wireless debugging (rtw89)"
+		depends on PACKAGE_kmod-rtw89
+		help
+		  Enable debugging output for rtw89 devices.
+
+	config PACKAGE_RTW89_DEBUGFS
+		bool "Enable rtw89 debugfs support"
+		select KERNEL_DEBUG_FS
+		depends on PACKAGE_kmod-rtw89
+		help
+		  Select this to see extensive information about
+		  the internal state of rtw89 in debugfs.
+
+	config PACKAGE_RTW89_DEBUGMSG
+		bool "Realtek rtw89 debug message support"
+		depends on PACKAGE_kmod-rtw89
+		help
+		  Enable debug message support.
+endef
+
+define KernelPackage/rtw89
+  $(call KernelPackage/mac80211/Default)
+  TITLE:=Realtek RTW89 core
+  DEPENDS+= +kmod-mac80211
+  FILES:=$(PKG_BUILD_DIR)/drivers/net/wireless/realtek/rtw89/rtw89_core.ko
+  AUTOLOAD:=$(call AutoProbe,rtw89_core)
+  HIDDEN:=1
+endef
+
+define KernelPackage/rtw89-pci
+  $(call KernelPackage/mac80211/Default)
+  TITLE:=Realtek RTW89 PCI chips support
+  DEPENDS+= +@PCI_SUPPORT +kmod-rtw89
+  FILES:=$(PKG_BUILD_DIR)/drivers/net/wireless/realtek/rtw89/rtw89_pci.ko
+  AUTOLOAD:=$(call AutoProbe,rtw89_pci)
+  HIDDEN:=1
+endef
+
+define KernelPackage/rtw89-8851be
+  $(call KernelPackage/mac80211/Default)
+  TITLE:=Realtek RTL8852BE support
+  DEPENDS+= +@DRIVER_11AX_SUPPORT +kmod-rtw89-pci +rtl8851be-firmware
```

Comment:
Please select `+@DRIVER_11AX_SUPPORT` here too. The `DRIVER_11AX_SUPPORT` does not select it automatically.

Reply:
you means `+@DRIVER_11AC_SUPPORT` ?  
E.g.
`DEPENDS+= +@DRIVER_11AC_SUPPORT  +@DRIVER_11AX_SUPPORT +kmod-rtw89-pci +rtl8851be-firmware`

Reply:
sorry, yes you are right

Reply:
adding it to `rtw89` it probably the best solution. I assume they will all support AC or later.

Reply:
OK I  added `+@DRIVER_11AC_SUPPORT  +@DRIVER_11AX_SUPPORT` to rtw89 and tested functionality.